com.unity.render-pipelines.universal/Runtime/XR/XRPass.cs

```diff
@@ -452,8 +455,34 @@ internal void UpdateGPUViewAndProjectionMatrices(CommandBuffer cmd, ref CameraDa
                     stereoProjectionMatrix[i] = GL.GetGPUProjectionMatrix(stereoCameraProjectionMatrix[i], isRenderToTexture);
                 }
                 RenderingUtils.SetStereoViewAndProjectionMatrices(cmd, stereoViewMatrix, stereoProjectionMatrix, stereoCameraProjectionMatrix, true);
+                if (cameraData.xr.canMarkLateLatch)
+                    MarkLateLatchShaderProperties(cmd, ref cameraData);
```

Comment:
Late latching uses the current stereo projection matrix as is and doesn't modify it (only view and world matrices are updated). Thus, if the stereo projection is y-flipped coming out of GL.GetGPUProjectionMatrix(), it will simply pass through to late latching as is (see line 474 of XRPass.cs SetLateLatchProjectionMatrices()). Render to texture would also not be affected.